src/Strategy/Node/Pnpm/PnpmLock.hs

```diff
@@ -76,17 +77,55 @@ import Path (Abs, File, Path)
 --          - For dependency resolved via git resolver, format is: "${Url}".
 --          - For dependency resolved via directory resolver, format is: "file:${relativePath}".
 --
+--
+--  Pnpm lockfile (v6) differs (v5), in following manner:
+--  -----------------------------------------------------
+--
+--    * `importers` shape merges specifiers and version, in singular object:
+--      @
+--      > importers:
+--      >    dependencies:
+--      >      aws-sdk:
+--      >        specifier: 2.1148.0
+--      >        version: 2.1148.0
+--      @
+--
+--    * Key of `packages` refer (e.g. "/buffer@4.9.2") denotes name of dependency and resolved version using '@' separator
+--        - For dependency resolved via registry resolver, format is: "/${dependencyName}@${resolvedVersion}${peerDepsInBrackets}".
+--      @
+--      >   /ieee754@1.1.13:
+--      >      resolution: {integrity: sha512...}
+--      >      dev: false
+--      >
+--      >   /@clerk/nextjs@4.22.1(next@13.4.10)(react-dom@18.2.0)(react@18.2.0):
+--      >       resolution: {integrity: sha512...}
+--      @
+--
+--    * If project has set peerDependencies to be not auto installed, pnpm
+--      by default, does not include them in the lockfile. So, no additional
+--      work is required for newly introduced `settings.autoInstallPeers` field.
```

Comment:
I don't understand what this has to do with our usage. Can you clarify?

Specifically:
- Does FOSSA CLI need to do anything special to correctly report peer deps?
- If not, why not? If so, why so, and what does it need to do?